app/scaffold-controllers/welcome-screen/welcomeHeaderController.js

```diff
@@ -0,0 +1,65 @@
+define([
+    'bluebird',
+    'plugins/headerBarPlugin',
+    'config/welcomeConfig'
+],
+/* eslint-disable */
+function(
+    Promise,
+    HeaderBarPlugin,
+    WelcomeConfig
+) {
+/* eslint-enable */
+
+    var WelcomeHeaderController = function(headerBar) {
+        this.viewPlugin = headerBar;
+    };
+
+    WelcomeHeaderController.init = function() {
+        return HeaderBarPlugin.init().then(function(headerBar) {
+            headerBar.setTextColor(WelcomeConfig.colors.textColor);
+            headerBar.setBackgroundColor(WelcomeConfig.colors.backgroundColor);
+            headerBar.hideBackButtonText();
+
+            headerBar.setCenterTitle(
```

Comment:
Does this title persist during navigations? What would you think about adding it to the header content (`generateContent`) structure?